### PR TITLE
 EOS-21183:Rename sspl-ll service to sspl

### PR DESCRIPTION
**Pull request by shriya-deshmukh**
## Problem Statement:

Rename sspl-ll service name to sspl.

## Solution Design:

- Changed 'sspl-ll.service' to 'sspl.service'.
- Changed daemon file name from sspl_ll_d to sspl_d.

## Coding:

* [x] Did you follow coding standard and verified it with flake8? 

* [x] Did your address all Codacy issues?

## Testing:

* [ ] Unittest updated and all unittests are passing?

* [ ] Sanity/Self tests are updated (if applicable)?

* [ ] Manual testing done covering happy path and non-happy path?

## Integration:

* [ ] If there is any interface change, did you communicate it to other components?
* [ ] If there is any interface change, other componenent gate gatekeepers are ready to accept the change?
* [ ] Did you complete deployment test if applicable?

## PR checklist:
* [x] Did you add Jira number to the PR?

* [x] DCO and cla-signed?

-----
[View rendered cortx-monitorQuickstartGuide.md](https://github.com/shriya-deshmukh/cortx-monitor/blob/EOS-21183/cortx-monitorQuickstartGuide.md)

### CODE REVIEW

File: low-level/files/etc/rsyslog.d/1-ssplfwd.conf [hunk 1]
```diff
@@ -30,10 +30,10 @@ $UDPServerRun 514
 
 # Enables log redirection for SSPL
 
-set $.service = "sspl-ll";
+set $.service = "sspl";
 
 # programname specifies the static part of the tag which is defined by syslog.
-# Ex: in "sspl-ll[12345]", programname will be sspl-ll.
+# Ex: in "sspl[12345]", programname will be sspl.
```
Comment (mariyappanp) [inline low-level/files/etc/rsyslog.d/1-ssplfwd.conf:36]: Correct the word, programname.

**Review by mariyappanp — COMMENTED** [review]
Ensure below.
1. Other component adapt this change
2. Deploy whole cortx stack and check after that and overall hctl/pcs status is good.

**Comment by cdeshmukh** [comment]
> Ensure below.
> 
> 1. Other component adapt this change
> 2. Deploy whole cortx stack and check after that and overall hctl/pcs status is good.

we are delivering this in feature branch to allow other components to adopt first & confirm that it works,

File: low-level/files/opt/seagate/sspl/setup/sspl_post_install.py [hunk 2]
```diff
@@ -341,20 +341,20 @@ def install_sspl_service_files(self):
...
-            os.symlink("%s/conf/sspl-ll.service.LR2" % (consts.SSPL_BASE_DIR),
+            os.symlink("%s/conf/sspl.service.LR2" % (consts.SSPL_BASE_DIR),
                        currentProduct)
-        shutil.copyfile(currentProduct, "/etc/systemd/system/sspl-ll.service")
+        shutil.copyfile(currentProduct, "/etc/systemd/system/sspl.service")
 
     def enable_sspl_service(self):
-        """Enable sspl-ll service."""
-        self.dbus_service.enable("sspl-ll.service")
+        """Enable sspl-d service."""
```
[the remark this answers is not in the document]
Comment (shriya-deshmukh) [reply low-level/files/opt/seagate/sspl/setup/sspl_post_install.py]: done.

File: low-level/files/etc/rsyslog.d/1-ssplfwd.conf (hunk 1 above)
Comment (shriya-deshmukh) [reply low-level/files/etc/rsyslog.d/1-ssplfwd.conf]: Done.

**Review by mariyappanp — APPROVED** [review]
LGTM.

File: installation/deps/sspl-ll.service.d/test-overrides.conf [hunk 3]
```diff
@@ -18,6 +18,6 @@ Wants=rabbitmq-server.service
 After=rabbitmq-server.service syslog.target
 
 [Service]
-ExecStart=/root/low-level/sspl_ll_d -c /etc/sspl.conf
+ExecStart=/root/low-level/sspl_d -c /etc/sspl.conf
```
Comment (cdeshmukh) [inline installation/deps/sspl-ll.service.d/test-overrides.conf:21]: need to be sspld

**Review by cdeshmukh — APPROVED** [review]

**Pull request merged**